packages/generator/src/service/index-file.ts

```diff
@@ -21,11 +21,9 @@ export function indexFile(service: VdmServiceMetadata): SourceFileStructure {
         exportStatement(complexType.typeName)
       ),
       ...service.enumTypes.map(enumType => exportStatement(enumType.typeName)),
-      ...(service.functionImports && service.functionImports.length
-        ? [exportStatement('function-imports')]
-        : []),
-      ...(service.actionImports && service.actionImports.length
-        ? [exportStatement('action-imports')]
+      ...((service.functionImports && service.functionImports.length) ||
+      (service.actionImports && service.actionImports.length)
+        ? [exportStatement('operations')]
```

Comment:
[req] You can simplify this statement as below. (But I would again expect `service.operations?.length` here.)
```suggestion
      ...(service.functionImports?.length || service.actionImports?.length)
        ? [exportStatement('operations')]
```

Reply:
Yes, I'll shorten this as well, although we don't need the conditional access on `service.functionImports` because its TS definition shows it always exists.